pipeline.yml

```diff
@@ -0,0 +1,47 @@
+version: 3.0.1
+pipeline_os: windows
+create_github_release: true
+create_pr_release_to_master: true
```

Comment:
What is this for?

Reply:
create_pr_release_to_master makes a pull request with a version bump when manually run on release branches: `release-patch`, `release-minor`, `release-major`. This is how GD repos manage their versioning

Reply:
e.g. https://git.autodesk.com/aecgd/RefineryClient/pull/1769